templates/logsearch-infrastructure-warden.yml

```diff
@@ -83,417 +83,417 @@ networks_default:
   subnets:
   - cloud_properties:
       name: random
-    range: 10.244.0.0/30
```

Comment:
Makes sense - hadn't seen a reference to this in the commits and thought it might be unintentionally included.